site/src/pages/release-notes/1.17.0.mdx

```diff
@@ -0,0 +1,309 @@
+  when <type://RequestContextExportingAppender> is used. #4285
+- You no longer see the `InputStream` returned by <type://StreamMessage#toInputStream(Function)> blocks
+  indefinitely in a certain situation. #4268 #4271
+- A <type://WriteTimeoutException> is now thrown when a header only request times out. #4255 #4259
+- The request now completes even when the content sanitizer throws an exception. #4248
+- A proxy client doesn't resolve a hostname anymore. #4244 #4245
+- The Spring Boot integration module doesn't create a new <type://WebClient> anymore every time
+  it's injected. #4240
+- Armeria server sends now 400 Bad Request when an HTTP/2 upgrade request contains invalid headers. #4016 #4224
+- <type://EurekaUpdatingListener> now registers itself with the `instanceId` which is `hostname:appname:port`,
+  so it doesn't cause a conflict when more than one app with the same name are running in the same host. #4223
+- A ramping up <type://EndpointSelectionStrategy> now works correctly in combination with
+  <type://HealthCheckedEndpointGroup>. #4221
+- The information of a thrown exception in gRPC service is now propagated to the client via the
+  `grpc-status-details-bin` header. #4203 #4204
+
+## 🏚️ Deprecations
+
+- <type://AbstractEndpointSelector#select(ClientRequestContext,ScheduledExecutorService,long)?full>
+  is deprecated in favor of
+  <type://AbstractEndpointSelector#select(ClientRequestContext,ScheduledExecutorService)?full>. #4246
+- <type://ThriftSerializationFormats#protocolFactory(SerializationFormat)?full> is deprecated in favor of
+  <type://ThriftSerializationFormats#protocolFactory(SerializationFormat,int,int)?full>.
+  #4226
+- `BINARY`, `COMPACT`, `JSON`, `TEXT`, and `TEXT_NAMED_ENUM` in <type://ThriftProtocolFactories> are deprecated
+  in favor of corresponding factory methods. #4226
+- <type://StreamMessage#of(Path,int)?full>, <type://StreamMessage#of(Path,ByteBufAllocator,int)?full> and
+  <type://StreamMessage#of(Path,ExecutorService,ByteBufAllocator,int)?full> are deprecated
+  in favor of <type://StreamMessage#builder(Path)?full>. #4058
+- <type://ServiceConfig#shutdownBlockingTaskExecutorOnStop()>,
+  <type://ServiceConfig#shutdownAccessLogWriterOnStop()>,
+  <type://ServerConfig#shutdownBlockingTaskExecutorOnStop()> and
+  <type://ServerConfig#shutdownWorkerGroupOnStop()> are not used anymore. #4280
+
+## ☢️ Breaking changes
+
+- <type://ByteStreamMessage> is returned instead of `StreamMessage<HttpData>`. #4058
+  - e.g <type://StreamMessage#of(File)?full>, <type://SplitHttpMessage#body()>,
+    <type://Multipart#toStreamMessage()> and <type://BodyPart#content()>
+- `HttpService.exchangeType(RequestHeaders, Route)` is now
+  <type://HttpService#exchangeType(RoutingContext)>. #4177
+
+## ⛓ Dependencies
+
+- Brave 5.13.8 → 5.13.9
+- Bucket4J 7.4.0 → 7.5.0
+- Dropwizard metrics 4.2.9 → 4.2.10
+- GraphQL 1.8.0 → 1.8.2
+- gRPC-Java 1.45.1 → 1.47.0
+- grpc-Kotlin 1.2.1 → 1.3.0
```

Comment:
```suggestion
- gRPC-Kotlin 1.2.1 → 1.3.0
```

Reply:
Don't need CI builds for this change. It could be applied after CI build passes.